src/node-red/integration-technologies/index.md

```diff
@@ -0,0 +1,12 @@
+---
+eleventyNavigation:
+  key: "Integration Technologies"
+meta:
+   title: Using Different Technologies for Building Applications with Node-RED.
+   description: Learn how to leverage various integration technologies with Node-RED for building robust and interconnected applications.
+   keywords: node-red, flowfuse, node-red integration, integration technologies, webhook, rest api
+---
+
+# Integration Technologies
```

Comment:
What else is going to be in this category?

Reply:
we need write content on GraphQL , ETL

Reply:
If we name the category "Data Integration," then we can also include a Kafka blog here? but we will loose an opportunity to create one more category called "streaming platforms"